CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v1.14.0-dev
+
+#### Enchancements
+
+- Add support for custom APIs [#149](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/149)
```

Comment:
Just being nit picky here. But it might be preferential to be specific here, that we added a helper to use to calling custom api's. Otherwise people might assuming we did more and wonder why the client doesn't have specific API classes for their environments custom api's.